sunkit_spex/models/physical/tests/test_thermal.py

```diff
@@ -49,62 +49,68 @@ def fvth_simple():
     temperature = 6 * u.MK
     emission_measure = 1e44 / u.cm**3
     abundance_type = DEFAULT_ABUNDANCE_TYPE
-    relative_abundances = None
     observer_distance = (1 * u.AU).to(u.cm)
-    inputs = (energy_edges, temperature, emission_measure, abundance_type, relative_abundances, observer_distance)
-    ssw_output = [
-        0.49978617,
-        0.15907305,
-        0.052894916,
-        0.017032871,
-        0.0056818775,
-        0.0019678525,
-        0.00071765773,
-        0.00026049651,
-        8.5432410e-05,
-        3.0082287e-05,
-        1.0713027e-05,
-        3.8201970e-06,
-        1.3680836e-06,
-        4.9220034e-07,
-        1.7705435e-07,
-        6.4002016e-08,
-        2.3077330e-08,
-        8.3918881e-09,
-        3.0453193e-09,
-        1.1047097e-09,
-        4.0377532e-10,
-        1.4734168e-10,
-        5.3671578e-11,
-        1.9628120e-11,
-        7.2107064e-12,
-        2.6457057e-12,
-        9.6945607e-13,
-        3.5472713e-13,
-        1.3051763e-13,
-        4.8216642e-14,
-        1.7797136e-14,
-        6.5629896e-15,
-        2.4178513e-15,
-        8.8982728e-16,
-        3.2711010e-16,
-        1.2110889e-16,
-        4.4997199e-17,
-        1.6709174e-17,
-        6.2011332e-18,
-        2.2999536e-18,
-        8.5248536e-19,
-        3.1576185e-19,
-        1.1687443e-19,
-        4.3226546e-20,
-        1.5974677e-20,
-        5.9133793e-21,
-        2.2103457e-21,
-        8.2594126e-22,
-        3.0853276e-22,
-        1.1521560e-22,
-    ] * SSW_INTENSITY_UNIT
-    return inputs, ssw_output
+    mg, al, si, s, ar, ca, fe = 8.15, 7.04, 8.1, 7.27, 6.58, 6.93, 8.1
+    inputs = (energy_edges, temperature, emission_measure, mg, al, si, s, ar, ca, fe, abundance_type)
+    inputs_class = (temperature, emission_measure, mg, al, si, s, ar, ca, fe, abundance_type)
+
```

Comment:
Might want to turn off autofarmatting for these you can use `# fmt: off` and `# fat: on` before and after block to allow you to format as you please maybe joining the line and wrapping at the line lenght 120 or something?